nemo/backends/pytorch/actions.py

```diff
@@ -232,7 +232,10 @@ def __setup_optimizer(
                 )
             elif optimizer_class.lower() == "adam":
                 optimizer = optim.Adam(
-                    params=params_to_optimize, lr=lr, betas=optimization_params.get("betas", (0.9, 0.999)),
+                    params=params_to_optimize,
```

Comment:
Would you please add eps and amsgrad params too?

Reply:
The same for adam_w.

Reply:
done